apps/asctb-reporter/src/app/components/home/home.component.html

```diff
@@ -38,6 +47,8 @@
               class="section-selector"
               [class.section-selected]="$index === selectedVideoSection()"
               hraButtonVariant="primary"
+              hraFeature="video-section-{{ section.sec }}"
```

Comment:
Update to `[hraFeature]="section.sec | slugify"` (make sure to merge base branch to access `slugify` pipe)